bucket4j-spring-boot-starter-context/src/main/java/com/giffing/bucket4j/spring/boot/starter/context/properties/Bucket4JBootProperties.java

```diff
@@ -43,6 +43,20 @@ public class Bucket4JBootProperties {
      */
     private String cacheToUse;
 
+    /**
+     * Defines which type of URL pattern parser will be used to match incoming request paths and queries.
+     *
+     * <p>Available options:</p>
+     * <ul>
+     *     <li><b>regex</b> — uses regular Java regular expressions (e.g., <code>^/api/.*</code>)</li>
+     *     <li><b>path-pattern</b> — uses Spring-style path patterns (e.g., <code>/api/**</code>)</li>
+     * </ul>
+     *
+     * <p>Default is <b>regex</b>.</p>
+     * <p>This setting controls how URL patterns in Bucket4j filters will be interpreted.</p>
+     */
+    private String urlPatternParser = "regex";

```

Comment:
Please use an enum again. If someone misspells it, the application will only fail at startup due to the missing UrlPatternParser bean, and the user won't know how to resolve the issue.